sonaranalyzer-dotnet/build/create-issue.ps1

```diff
@@ -100,7 +100,8 @@ function New-Issue() {
         -Uri $projectUri `
         -Method "POST" `
         -Headers $headers `
-        -Body ($payload | ConvertTo-Json)
+        -Body ($payload | ConvertTo-Json) `
+        -UseBasicParsing

```

Comment:
Yep I read the documentation I was just curious to know what was the motivation behind this change.